ui/raidboss/data/04-sb/raid/o4s.js

```diff
@@ -1035,72 +1035,138 @@
+        'Black Hole': 'Schwarzes Loch',
         'Engage!': 'Start!',
         'Exdeath': 'Exdeath',
+        'uses Almagest': 'uses Almagest', // FIXME
+        'uses Dualcast': 'uses Dualcast', // FIXME
+        'uses Emptiness': 'uses Emptiness', // FIXME
       },
       'replaceText': {
+        '--Acceleration Bomb Resolves--': '--Acceleration Bomb Resolves--', // FIXME
+        '--Allagan Field Explodes--': '--Allagan Field Explodes--', // FIXME
+        '--Boss Targetable--': '--Boss Targetable--', // FIXME
+        '--Boss Untargetable--': '--Boss Untargetable--', // FIXME
+        '--LB Gauge Resets--': '--LB Gauge Resets--', // FIXME
         '--targetable--': '--anvisierbar--',
         '--untargetable--': '--nich anvisierbar--',
+        'Aero III': 'Windga',
+        'Almagest': 'Almagest',
         'Black Hole': 'Schwarzes Loch',
         'Black Spark': 'Schwarzer Funke',
         'Blizzard III': 'Eisga',
+        'Charge': 'Sturm',
+        'Charybdis': 'Charybdis',
         'Clearout': 'Kreisfeger',
         'Collision': 'Aufprall',
+        'Cursed Shriek': 'Schrei der Verwünschung',
+        'Delta Attack': 'Delta-Attacke',
         'Doom': 'Verhängnis',
+        'Double Attack': 'Doppelangriff',
+        'Double Attack Tethers': 'Double Attack Tethers', // FIXME
+        'Dualcast': 'Doppelzauber',
+        'Earth Shaker': 'Erdstoß',
+        'Emptiness': 'Tobende Leere',
         'Enrage': 'Finalangriff',
+        'Final Battle': 'Final Battle', // FIXME
         'Fire III': 'Feuga',
         'Flare': 'Flare',
+        'Flood of Naught': 'Flut der Leere',
+        'Forked Lightning': 'Gabelblitz',
+        'Frenzied Fist': 'Rasende Faust',
+        'Frenzied Sphere': 'Rasender Orbis',
+        'Grand Cross Alpha': 'Supernova Alpha',
+        'Grand Cross Delta': 'Supernova Delta',
+        'Grand Cross Omega': 'Supernova Omega',
+        'HP Down Debuff': 'HP Down Debuff', // FIXME
         'Holy': 'Sanctus',
-        'Meteor': 'Meteor',
+        'Knockback': 'Knockback', // FIXME
+        'Light and Darkness': 'Licht und Dunkelheit',
+        'Meteor': 'Meteo',
+        'Neverwhere': 'Nirgendwann',
+        'Random Elemental': 'Random Elemental', // FIXME
         'The Decisive Battle': 'Entscheidungsschlacht',
         'Thunder III': 'Blitzga',
         'Vacuum Wave': 'Vakuumwelle',
+        'Water': 'Aqua',
+        'White Hole': 'Weißes Loch',
         'Zombie Breath': 'Zombie-Atem',
       },
       '~effectNames': {
         'Deep Freeze': 'Tiefkühlung',
         'Doom': 'Verhängnis',
         'Lightning Resistance Down': 'Blitzresistenz -',
         'Paralysis': 'Paralyse',
-        'Pyretic': 'Pyretisch',
+        'Pyretic': 'Hitze',
         'Zombification': 'Zombie',
       },
     },
     {
       'locale': 'fr',
       'replaceSync': {
-        'Black Hole': 'Trou Noir',
-        'Engage!': 'À l\'attaque',
+        'Black Hole': 'Trou noir',
+        'Engage!': 'À l\'attaque !',
```

Comment:
I am a little bit skeptical about you adding this in everywhere.  I think it would be better to not have it.  My memory (maybe wrong?) is that this isn't a space character at the end, and so it's very easy to get wrong.

Reply:
In the game files it is a `<Indent/>` 
This is just to make them all be the same (see merged arr ones)
In my ARR PR, there was also the topic of my script not handling the indent very well.
So want me to replace all of them?

Reply:
the indent issue is also the part of #990 
is there a known rule how the indent is processed by the game?

Reply:
Sorry to be forgetful, but I forgot that ACT strips the whitespace before the exclamation (or did in the past).  So, technically `À l\'attaque!` is correct.  However, I wonder if it would be safer and more future proof to just drop the space and exclamation point in case this ever changes in ACT.

See this huge thread: https://github.com/quisquous/cactbot/issues/57#issuecomment-371376181

Reply:
for sb content i removed the ` !` now.